packages/seacas/libraries/ioss/src/Ioss_FileInfo.C

```diff
@@ -196,10 +193,15 @@ namespace Ioss {
         IOSS_ERROR(errmsg);
       }
       free(path);
-      return (stat_fs.f_type == NFS_FS);
+
+#if defined(__APPLE__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)
+      return (0 == ::strcmp("nfs", stat_fs.f_fstypename));
+#else
+      /* linux statfs defines that 0x6969 is NFS filesystem */
+      return (stat_fs.f_type == 0x6969);
```

Comment:
I would prefer that the `#define NFS_FS` be used here.  Many static analyzers complain about magic numbers.  I realize that the define would be used in exactly one place and that there are many other magic numbers throughout the code base, but ...